apps/plasmo/src/components/Calendar/DayIndex.tsx

```
@@ -0,0 +1,21 @@
+import React from "react"
+
+const NepaliWeekdays = () => {
+  const weekdays = ["आ", "सो", "मं", "बु", "बि", "शु", "श"]
+
+  return (
+    <div className="plasmo-bg-slate-900 plasmo-w-full plasmo-py-3">
+      <div className="plasmo-flex plasmo-justify-between plasmo-items-center plasmo-px-4">
+        {weekdays.map((day, index) => (
+          <div
+            key={index}
+            className="plasmo-text-gray-300 plasmo-text-sm plasmo-font-hindi">
```

Biome

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)